engine/classes/Elgg/Application/CacheHandler.php

```diff
@@ -18,28 +18,32 @@ class CacheHandler {
 	private $application;
```

Comment:
FYI changes here just mirror the I18nHandler, so we can later refactor these in a similar way.